orga/app/services/session.js

```diff
@@ -19,10 +13,10 @@ export default class CurrentSessionService extends SessionService {
   routeAfterAuthentication = 'authenticated';
 
   async handleAuthentication() {
-    const isFranceDomain = this.currentDomain.isFranceDomain;
     await this.currentUser.load();
-    const userLocale = this.currentUser.prescriber.lang;
-    await this.handleLocale({ isFranceDomain, userLocale });
+
+    const language = this.currentUser.prescriber.lang;
+    await this.locale.detectBestLocale({ language, user: this.currentUser.prescriber });
```

Comment:
Ici, on a pas besoin de passer `language`, le `prescriber.lang` sera utilisé dans le `detectBestLocale`:
```suggestion
    await this.locale.detectBestLocale({ user: this.currentUser.prescriber });
```